### DISCUSSION TITLE
How to mandate pinning via included presets or shared local presets

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#14120] [Request Help] [answered]
Hi all

Recently I've noticed a handful of my repo `package.json` files (both on GitHub and in Bitbucket Cloud) have had dependencies become unpinned for one reason or another, and that Renovate has not suggested that these should be re-pinned and/or that newer dependency versions are available

Is there a way in the configuration to enforce that any future updates to the `package.json` will check to see if anything new needs to be pinned or re-pinned?

**Comment by rarkins** [comment, marked as answer]
rangeStrategy=pin
